packages/react-components/react-nav-preview/src/components/NavSubItemGroup/renderNavSubItemGroup.tsx

```diff
@@ -10,6 +10,5 @@ import type { NavSubItemGroupState, NavSubItemGroupSlots } from './NavSubItemGro
 export const renderNavSubItemGroup_unstable = (state: NavSubItemGroupState) => {
   assertSlots<NavSubItemGroupSlots>(state);
 
-  // TODO Add additional slots in the appropriate place
-  return <state.root />;
+  return state.open ? <state.root>{state.root.children}</state.root> : null;
```

Comment:
I think the new slots config takes care of the props iirc
```suggestion
  return <state.root />;
```

Reply:
Stuck with the way accordion did it, and mocked the conformance tests to allow it to pass.